### ISSUE TITLE
GUI or software to test out this new AI technology

### ISSUE THREAD

**Issue by thenormal** [facebookresearch/demucs#216]
## ❓ Questions

Hello and congrats on the great work on Demucs. I was wondering if there are any plans for the near future in terms of developing a GUI or an installable software in order to test out this new AI technology.

Keep up the great work.

**Comment by adefossez** [comment]
Thank you @thenormal . as much as I would want to provide a simple to use GUI for this project, this is something I would have to do on my free time, as it doesn't exactly fit my mission as a scientist at FAIR. There is a number of challenges to overcome, in particular to provide a good and consistent experience across all OSes. While I definitely have it somewhere in my head, I don't think this is something I would do in the near future.

**Comment by thenormal** [comment]
> Thank you @thenormal . as much as I would want to provide a simple to use GUI for this project, this is something I would have to do on my free time, as it doesn't exactly fit my mission as a scientist at FAIR. There is a number of challenges to overcome, in particular to provide a good and consistent experience across all OSes. While I definitely have it somewhere in my head, I don't think this is something I would do in the near future.

Fair enough. I'm looking forward to having this technology easily available to the masses, so people like me who don't know anything about coding can check it out. Best of luck with this project.

**Comment by AnshulJ999** [comment]
Would it be possible for you to work with the creator of SpleeterGUI? https://github.com/boy1dr/SpleeterGui
He had expressed interest in adding Demucs to his software too. The GUI is already well-functioning and pretty easy to use. Would be worth considering.

**Comment by CarlGao4** [comment]
@thenormal [Demucs GUI](https://carlgao4.github.io/demucs-gui) is available.
